stackit/internal/services/objectstorage/objectstorage_acc_test.go

```diff
@@ -107,19 +140,34 @@ func TestAccObjectStorageResource(t *testing.T) {
 						"data.stackit_objectstorage_bucket.bucket", "url_virtual_hosted_style",
 					),
 
-					// Credentials group data
-					resource.TestCheckResourceAttr("data.stackit_objectstorage_credentials_group.credentials_group", "project_id", credentialsGroupResource["project_id"]),
+					// Credential group data
```

Comment:
I think here it's fine to leave it plural (as it os on line 83) as it's the "credentials group"